lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -71,6 +71,9 @@ class ReportUIFeatures {
    * @param {ReportJSON} report
    */
   initFeatures(report) {
+    // Quit if it's a devtools report.
+    if (this._dom.document().querySelector('.lh-devtools')) return;
```

Comment:
WDYT about a universal `isDevTools` or something at the beginning rather than check this in multiple places?

Reply:
done